### DISCUSSION TITLE
Start time for brews seem wrong?

### DISCUSSION THREAD

**Discussion by aeroevan** [graphefruit/Beanconqueror#408] [General]
My brew time seems to start at some random(ish) time. Is there a timezone setting or a way to get it to start from 0?

![file](https://user-images.githubusercontent.com/44744/192125080-82599750-6c16-4351-a31d-45a5738fffd1.jpeg)

**Comment by graphefruit** [comment]
Hi @aeroevan, 
sorry for the late reply.
Can you make sure that the brew-time is "0" before starting?
Also, try to press the reset-button before starting and have a look if this will solve this issue.

In the next version (6.2) I've added some more settings to say when the scale should be resetted and when not - maybe this will help.

I'll let this ticket open here, so you can check with the next upcoming version.
Best regards & Sorry for the late reply
Lars

**Comment by aeroevan** [comment]
Thanks, pressing the reset button gets it to start from 0 again. I assumed it would "reset" for each brew, but maybe not. The brew time on my scale (lunar 2021) read 0 at the start each time.

Tried out a blooming profile today, but seemed to work as expected after pressing the reset button.

![file (1)](https://user-images.githubusercontent.com/44744/193410407-bea047c8-37c4-47bc-9eba-20e6de29a300.jpeg)

**Reply by graphefruit** [reply]
> Thanks, pressing the reset button gets it to start from 0 again. I assumed it would "reset" for each brew, but maybe not. The brew time on my scale (lunar 2021) read 0 at the start each time.
> 
> Tried out a blooming profile today, but seemed to work as expected after pressing the reset button.
> 
> ![file (1)](https://user-images.githubusercontent.com/44744/193410407-bea047c8-37c4-47bc-9eba-20e6de29a300.jpeg)

Can you confirm the brew time was "0"? 
The app has its own brew-time which it counts up.
When the brewtime says "87" it means 87 seconds, which would then be the starting point in the app for the flow.
Your 8776 seems realy high, because it means 146 minutes.

The timer from the scale is never used, just started and resetted.

**Reply by aeroevan** [reply]
The graph started when I told it to start, which is why I was so confused. I just pressed reset on each of my brews today and it seems to start from 0 each time. I could try not resetting tomorrow and see what happens.

**Reply by graphefruit** [reply]
@aeroevan In the next version, you cant start the timer when a time is still preset and you'll get a popup mentioning you need to reset.
Therefore the issue should be solved.
